Samples/Samples/Converters/EndTypeToBoolConverter.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Samples.ViewModel;
+using Xamarin.Forms;
+
+namespace Samples.Converters
+{
+    public class EndTypeToBoolConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            if (value == null || !(value is string type))
+            {
+                return false;
+            }
+            switch (type)
+            {
+                case RecurrenceEndType.Indefinitely:
```

Comment:
group cases that have the same return value

Reply:
Altered in latest commit